app/models/concerns/visible.rb

```diff
@@ -0,0 +1,24 @@
+module Visible
+
+  #Validations are methods called at the class level
+  #this extend give us a simpler way to include them.
+
+  extend ActiveSupport::Concern
```

Comment:
@JuanMaGiraldo how the concerns work?

Reply:
Concerns are modules that extend from the `ActiveSupport::Concern` class. They are a useful way to reuse behaviors that can be repeated in both models and controllers. This helps to maintain the DRY principle.

It will also inherit the **included** method that will be executed within the context of the included class.